### PR TITLE
Looking at oxidized text, might be that my updated fw was already there. Wasn't sure before

### PR DESCRIPTION
**Pull request by patrickgilsf**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c [hunk 1]
```diff
@@ -0,0 +1,431 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+KEY22 -
+KEY23 -
+KEY24 -
+KEY25 - 
+KEY26 - If KEY25 is down, back to layer 1
+KEY27 - If KEY25 && KEY26 are down, puts it in setup mode
+*/
+
+//initialize booleans for stacked functions:
+bool kp_1 = false;
```
Comment (drashna) [inline keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c:97]: you could use `bool key[26] = {0};` for this instead.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c (hunk 2 above)
Comment (patrickgilsf) [reply keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c:100]: Thanks for the advice. I'm not that familiar with C, mostly JS and node.

File: keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c (hunk 1 above)
Comment (patrickgilsf) [reply keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c:4]: I have added the header and push the changed to the repo. Please let me know if there are nay issues.

File: keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c (hunk 2 above)
Comment (patrickgilsf) [reply keyboards/ymdk/ymd09/keymaps/patrickgilsf/keymap.c:100]: @drashna are my changes ok in the new push? Will this FW get merged at any point? TY.

**Pull request closed without merging**